indexed_gzip/zran_file_util.c

```diff
@@ -105,20 +105,41 @@ int64_t _ftell_python(PyObject *f) {
  */
 int _fseek_python(PyObject *f, int64_t offset, int whence) {
 
-    PyObject *data;
+    PyObject *data = NULL;
+    PyObject *seek_fn_name = NULL;
+    PyObject *whence_ = NULL;
+    PyObject *offset_ = NULL;
 
     _ZRAN_FILE_UTIL_ACQUIRE_GIL
 
-    data = PyObject_CallMethod(f, "seek", "(l,i)", offset, whence);
-    if (data == NULL)
+    // We can't use PyObject_CallMethod with multiple arguments
+    // because it causes tests with 32-bit OS wheels to fail
+    // (as the second argument is not passed correctly to Python),
+    // so we have to manually build up the arguments instead.
+    // TODO: File an issue with cpython about this.
+    // if ((data = PyObject_CallMethod(f, "seek", "(l,i)", offset, whence)) == NULL)
```

Comment:
I wonder if the problem is with the `l` in `(l, i)`? An `int64_t` is being passed which, on a 32 bit platform, may correspond to `long long`, and require `L`. I'll merge this in as is, but might play around to see if I can figure it out.